draft-ietf-quic-recovery.md

```diff
@@ -973,24 +936,20 @@ During recovery, the congestion window remains unchanged irrespective of new
 losses or increases in the ECN-CE counter.
 
 
-## Tail Loss Probe
-
-A TLP packet MUST NOT be blocked by the sender's congestion controller. The
-sender MUST however count TLP packets against bytes in flight, since a TLP adds
-network load without establishing packet loss.
+## Probe Timeout
 
-Acknowledgement or loss of tail loss probes are treated like any other packet.
+Probe packets MUST NOT be blocked by the congestion controller.  A sender MUST
+however count these packets as being additionally in flight, since these packets
+adds network load without establishing packet loss.  Note that sending probe
+packets might cause the sender's estimated bytes in flight to exceed the
+sender's congestion window until an acknowledgement is received that establishes
+loss or delivery of packets.
 
-## Retransmission Timeout
+A PTO expiration is classified as spurious or valid when an ACK frame is
+received that newly acknowledges packets in flight, see {{pto-loss}}.  On a
+valid PTO, the congestion window MUST be reduced to the minimum congestion
+window and slow start is re-entered.
```

Comment:
Just to confirm my understanding, this is a difference from TLP before, right? Before this change, if TLP triggered and you sent a probe packet that elicited an ACK for just that packet, you wouldn't go back into slow start, but now an ACK for only the first probe packet would trigger slow start. Correct?

Reply:
Agreed, this is a potentially substantial change.

Reply:
If you have a lot of tails(ie: HTTP traffic), then it doesn't take too many packet losses to end up hitting this.  TLP was targeted partially at cases like that, and I think you'll find changing it will produce measurable performance regressions in web traffic.  We can do something else in a follow-up, but as I said before, I'd prefer text saying SHOULD consider an acknowledgement of the first PTO as standard packet loss, not extreme congestion, and treat it as such.

Reply:
I'm going to simplify this further. I think that having a verification is unnecessary; the behavior ends up being the same whether a PTO is verified or not. I can retain this notion of reducing the cwnd differently for the first 2 PTOs and then collapsing it to min_cwnd after that.